src/System.AppContext/src/System.AppContext.csproj

```diff
@@ -10,6 +10,7 @@
     <!-- The following line needs to be removed once we have a targeting pack for 4.6.2 -->
     <TargetingPackNugetPackageId Condition="'$(TargetGroup)'=='net462'">Microsoft.TargetingPack.NETFramework.v4.6</TargetingPackNugetPackageId>
     <PackageTargetFramework Condition="'$(PackageTargetFramework)' == ''">netstandard1.5</PackageTargetFramework>
+    <ExcludeResourcesImport Condition="'$(IsPartialFacadeAssembly)'=='true'">true</ExcludeResourcesImport>
```

Comment:
I believe I've heard @weshaggard in the past suggest always including the explicit comparison.  If for no other reason than consistency with the other conditions, I think I'll leave it.